civis/tests/test_io.py

```diff
@@ -85,22 +86,25 @@ def setup_class(cls, *mocks):
             res.result()  # block
 
             # create an export to check get_url. also tests export_csv
-            with tempfile.NamedTemporaryFile() as tmp:
+            with TemporaryDirectory() as temp_dir:
+                fname = os.path.join(temp_dir, str(uuid.uuid4()))
                 sql = "SELECT * FROM scratch.api_client_test_fixture"
                 database = 'redshift-general'
-                result = civis.io.civis_to_csv(tmp.name, sql, database,
+                result = civis.io.civis_to_csv(fname, sql, database,
                                                polling_interval=POLL_INTERVAL)
                 result = result.result()
                 cls.export_url = result['output'][0]['path']
                 assert result.state == 'succeeded'
 
             cls.export_job_id = result.sql_id
 
+    @pytest.mark.skipif(sys.platform.startswith('win'))  # TODO
```

Comment:
Did you fix this test? Does it still need to be skipped in Windows?

Reply:
The test isn't fixed yet, and I plan to. I threw in this skipif thing to suppress for something else. We're _very_ close to getting the whole thing working. I'll tag you when the PR is ready fo review.

Reply:
Great work on this so far!